tests/app/Application/DataSources/WalletDataSourceTest.php

```diff
@@ -22,6 +23,33 @@ protected function setUp(): void
         });
     }
 
+    /**
+     * @test
+     */
+    public function whenCacheIsNotFullSavesWalletInCache()
```

Comment:
saveWalletWhenCacheIsNotFull??

To choose test names we must first think about what the test will do and then why or when.
<action><when/why>

saveWallet><WhenCacheIsNotFull

Reply:
Top!! Thanks ⭐